src/InboundServer/handlers.js

```diff
@@ -735,7 +735,7 @@ const postBulkQuotes = async (ctx) => {
  */
 const putBulkQuotesById = async (ctx) => {
     // publish an event onto the cache for subscribers to action
-    await ctx.state.cache.publish(`bulkQuotes_${ctx.state.path.params.ID}`, {
+    await ctx.state.cache.publish(`bulkQuote_${ctx.state.path.params.ID}`, {
```

Comment:
Why are we changing this?

And would it impact an existing deployment since we are changing the keys here?

*the Same comment applies elsewhere through the PR.

Reply:
This is a bug. The outbound handler subscribed to bulkQuote where as inbound handler is publishing to 'bulkQuotes' which is wrong channel.
As per this line https://github.com/mojaloop/sdk-scheme-adapter/blob/37b1b389a314e9cd53a4a8785a44d48547b6e920/src/InboundServer/handlers.js#L842
I changed that to right value and also it follows the same pattern as transfers.

It seems nobody is using bulk functionality of scheme adopter with all these broken things